src/classes/BDI_DataImport_BATCH.cls

```diff
@@ -1328,8 +1328,6 @@ public class BDI_DataImport_BATCH implements Database.Batchable<sObject> {
         // now update the Contacts to create their Affiliations
 	    // unfortunately, we must manually clear these static flags since we are doing 
```

Comment:
need to remove these two lines of comments, since BDI no longer needs to clear these affiliation flags.